### PR TITLE
Env var option as part of README

### PR DESCRIPTION
**Pull request by CrowdHailer**

### CODE REVIEW

**Comment by CrowdHailer** [comment]
Question. Is it possible to use this setup for all settings? I would like to use it for the port and servername aswell

**Comment by kdisneur** [comment]
Good spot! You are right, we should add this information in the README.

Really, we should add it because the library not only supports this feature for `username` and `password`, it already supports it for all configuration fields.

What do you think of adding a more general sentence in the README about the support of `{:system, "ENVIRONMENT_VARIABLE"}` and also one in [the documentation](https://github.com/fewlinesco/bamboo_smtp/blob/develop/lib/bamboo/adapters/smtp_adapter.ex#L2)? 😁

File: README.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ The package can be installed as:
     adapter: Bamboo.SMTPAdapter,
     server: "smtp.domain",
     port: 1025,
-    username: "your.name@your.domain",
-    password: "pa55word",
+    username: "your.name@your.domain", # or {:system, "SMTP_USERNAME"}
+    password: "pa55word", # or {:system, "SMTP_PASSWORD"}
```
Comment (kdisneur) [inline README.md:37]: Could you use `"SMTP_PASS"` here? Or, update `"SMTP_PASS"` in the [library documentation](https://github.com/fewlinesco/bamboo_smtp/blob/develop/lib/bamboo/adapters/smtp_adapter.ex#L16) to use `"SMTP_PASSWORD"`?

We can use `SMTP_PASSWORD`, the naming is better but we would just like to keep all the example consistent. :)

**Comment by CrowdHailer** [comment]
What do you think about that addition to the README

**Comment by kdisneur** [comment]
Thanks - looks good to me! 👍 

If you can also add this sentence to the library documentation and rebase your commits, we will be really happy to merge it

**Comment by CrowdHailer** [comment]
develop is the default branch on your repository?

rebasing won't do anything, unless I am missunderstanding

**Comment by kdisneur** [comment]
Sorry - you are right the default branch is develop. I meant squashing your commits.

**Comment by CrowdHailer** [comment]
There is a squash and merge option on PRs is there not.

**Comment by kdisneur** [comment]
That's right, I always forget about this feature :)

Anyway, we can't merge the PR right now because the build fails. Credo found this issue on your last commit:

```
Code Readability                                                              
┃ 
┃ [R] ↓ There should be no trailing white-space at the end of a line.
┃       lib/bamboo/adapters/smtp_adapter.ex:7:1 (Bamboo.SMTPAdapter)
```

**Comment by CrowdHailer** [comment]
It's a shame these tools can't make formatting changes. I normally just use the github web interface when contributing to documentation stuff.

**Comment by kdisneur** [comment]
Thanks for the Pull Request :)

**Pull request merged**